src/pages/userManagement/UserManagerCollaborationEditTable.jsx

```diff
@@ -20,26 +21,40 @@ export default function UserManagersCollaborationEditTable({
   collaborationLoading,
   collaborationError,
 }) {
+  const {
+    data: currentUserData,
+    isFetching: userDataFetching,
```

Comment:
why fetching for this one and loading for the other one?

Reply:
No reason. I was just keeping a pattern that was already there. Changed to loading to keep in line with most everything else. Done.